scripts/prepare_cluster.sh

```diff
@@ -65,6 +64,10 @@ FLUX_GIT_BRANCH=${FLUX_GIT_BRANCH:-main}
 FLUX_PATH=${FLUX_PATH:-clusters/my-cluster/flux-system}
 ROOTFS_SIZE=${ROOTFS_SIZE:-20}
 
+if [ $IS_IN_AZURE == "true" ]; then
+    AZURE_VXLAN_IPRANGE=${AZURE_VXLAN_IPRANGE:-10.0.98.0/24}
+fi
```

Comment:
`if` condition isn't needed.
```suggestion
AZURE_VXLAN_IPRANGE=${AZURE_VXLAN_IPRANGE:-10.0.98.0/24}
```